scripts/cmake/vcpkg_build_make.cmake

```
@@ -132,6 +101,7 @@ function(vcpkg_build_make)
                 endif()
                 set(SHORT_BUILDTYPE "-dbg")
                 set(CMAKE_BUILDTYPE "DEBUG")
+                set(PATH_SUFFIX_${BUILDTYPE} "/debug")

```

```suggestion
                set(PATH_SUFFIX "/debug")
```
iiuc, there's no need for `PATH_SUFFIX` to be split across configurations.